packages/ng/forms/phone-number-input/formatter.ts

```diff
@@ -0,0 +1,8 @@
+import { E164Number, NumberFormat } from './types';
+import { parsePhoneNumber } from 'libphonenumber-js';
+
+export class PhoneNumberFormatter {
```

Comment:
I would expose the function directly. Maybe we could call it `luParsePhoneNumber` to avoid conflict with auto-import

Reply:
What if we expose just a function that takes the number and the format? basically removing the class that hosts it.

I'm not a big fan of exposing the parse method with our own name because it'd just mean rebranding it basically while we could expose it as-is just like we do for types. But also, exposing a simple way to format without having to call the parsing by yourself seems nice to have, idk